pkg/application/interfaces.go

```diff
@@ -102,16 +102,22 @@ type globalConfigEntryRepository interface {
 	domainservice.GlobalConfigEntryRepository
 }
 
+// TODO: remove this when refactoring is done
 type doguConfigEntryRepository interface {
 	domainservice.DoguConfigEntryRepository
 }
 
+type doguConfigRepository interface {
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>type `doguConfigRepository` is unused
<!-- __reviewdog__:ChA3NDkyZmY5OGRlN2MyMjg0EgpjaGVja3N0eWxl -->